packages/app/src/components/Blank.tsx

```diff
@@ -0,0 +1,284 @@
+const svg_cy = `<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Nit: maybe this does not need the `tsx` extension: it's not actually using TSX. Things might be a tiny bit quicker, since it won't run through the TSX transform.